### ISSUE TITLE
User image must be in same resource group as deployment!??!?!

### ISSUE THREAD

**Issue by tfnaylor** [Azure/azure-quickstart-templates#506]
The 101-vm-from-user-image template shows how to utilize a custom image in a template. It appears, however, that the image must preexist in the resource group, which of course doesn't make sense because you are using the template to create the resource group in the first place. This seems like a terrible implementation. Is there some way around this? 

We want to have our user images in a separate storage account and just reference them in our resource manager templates. Is there some way to add them to a private gallery that is accessible to all our deployments?

**Comment by singhkay** [comment]
@tfnaylor You can create a VM in a different resource group from that of the storage account in which the image lives. 
E.g. I create a VM in RG3 and captured into SA3 in RG3. Then I created a VM in RG6 using the image that still lives in SA3 in RG3.

**Comment by tfnaylor** [comment]
When I tried this I got an error that said something about "bad gateway" and the deployment hung indefinitely. In any event, I have just decided to use Chef to provision my nodes so this functionality is no longer required in my case. I also heard a rumor that eventually the Marketplace will allow user-created, private resource definitions so I might come back to this at that point. 

Thanks for answering.
